SpatialGDK/Source/SpatialGDKFunctionalTests/Public/SpatialFunctionalTest.h

```diff
@@ -166,46 +165,12 @@ class SPATIALGDKFUNCTIONALTESTS_API ASpatialFunctionalTest : public AFunctionalT
 			  Category = "Spatial Functional Test")
 	FWorkerDefinition GetAllClients() { return FWorkerDefinition::AllClients; }
 
-	// # Actor Delegation APIs
-	// clang-format off
-	UFUNCTION(CrossServer, Reliable, BlueprintCallable, Category = "Spatial Functional Test",
-		meta = (ServerWorkerId = "1", ToolTip = "Allows you to delegate authority over this Actor to a specific Server Worker. \n\nKeep in mind that currently this functionality only works in single layer Load Balancing Strategies, and your Default Load Balancing Strategy needs to implement ISpatialFunctionalTestLBDelegationInterface."))
-	// clang-format on
-	void AddActorDelegation(AActor* Actor, int ServerWorkerId, bool bPersistOnTestFinished = false);
-
-	// clang-format off
-	UFUNCTION(CrossServer, Reliable, BlueprintCallable, Category = "Spatial Functional Test",
-		meta = (ToolTip = "Remove Actor authority delegation, making it fallback to the Default Load Balacing Strategy. \n\nKeep in mind that currently this functionality only works in single layer Load Balancing Strategies, and your Default Load Balancing Strategy needs to implement ISpatialFunctionalTestLBDelegationInterface."))
-	// clang-format on
-	void RemoveActorDelegation(AActor* Actor);
-
-	// clang-format off
-	UFUNCTION(BlueprintCallable, Category = "Spatial Functional Test",
-		meta = (ToolTip = "Check is the Actor has it's authority delegated to a specific Server Worker. \n\nKeep in mind that currently this functionality only works in single layer Load Balancing Strategies, and your Default Load Balancing Strategy needs to implement ISpatialFunctionalTestLBDelegationInterface."))
-	// clang-format on
-	bool HasActorDelegation(AActor* Actor, int& WorkerId, bool& bIsPersistent);
-
-	// # Actor Interest APIs
-	// clang-format off
-	UFUNCTION(BlueprintCallable, Category = "Spatial Functional Test", meta = (ServerWorkerId = "1",
-		ToolTip = "Allow Server Worker to always have interest in an Actor, even it if moves outside its interest area.\n\nNote that this only works if you are using USpatialFunctionalTestGridLBStrategy."))
-	// clang-format on
-	void AddActorInterest(int32 ServerWorkerId, AActor* Actor);
-
-	// clang-format off
-	UFUNCTION(BlueprintCallable, Category = "Spatial Functional Test", meta = (ServerWorkerId = "1",
-		ToolTip = "Counterpart to AddActorInterest(), so it can only remove interest added by the other function. There's no way to force remove interest if the underlying configuration in GDK is giving it.\n\nNote that this only works if you are using USpatialFunctionalTestGridLBStrategy."))
-	// clang-format on
-	void RemoveActorInterest(int32 ServerWorkerId, AActor* Actor);
-
```

Comment:
ATM removing all these APIs there's no more ways to handle this via the testing framework. Is it planned to have another alternative or how do we expose these to BPs?

Reply:
I moved the API to Utils/DebugInterface.h
I'm not opinionated on how we expose them, I removed the ones from the test framework because they had a different API from what I actually implemented.
We could wrap them again, if the test framework does a bit more around these.

Reply:
I somehow missed USpatialGDKDebugInterface exposed functions. Do these functions then appear anywhere, and is that ok?

Reply:
They do appear anywhere at the moment. I do not have an opinion on whether they should be methods or free functions.

Reply:
Yeah I'm of the opinion that the utility functions should only be called within a Function Test, so lets keep the interface to that functionality within this class.